t/op/split.t

```diff
@@ -70,9 +70,13 @@ is($cnt, scalar(@ary), "Check element count from previous test");
 # Can we do it with the empty pattern?
 $_ = join(':', split(//, '123', 2));
 is($_, '1:23', "Split into specified number of fields with empty pattern");
-@ary = split(//, '123', 2);
-$cnt = split(//, '123', 2);
-is($cnt, scalar(@ary), "Check element count from previous test");
+$_ = join(':', split(//, '123', 6));
+is($_, '1:2:3:', "Split with empty pattern and LIMIT > length");
+for (0..5) {
```

Comment:
This prompts me to realise 0 is special, should probably add a check above for '1:2:3' in that case.

LGTM either way.

Reply:
Thanks @hvds. I've added some checks for LIMIT == -1 and 0.